boards/shields/adafruit_winc1500/doc/index.rst

```diff
@@ -0,0 +1,70 @@
+.. _adafruit_winc1500:
+
+Adafruit WINC1500 Wifi Shield
+#############################
+
+Overview
+********
+
+The Adafruit WINC1500 Wifi shield is an Arduino
+compatible shield based on the ATWINC1500 wifi controller
+from Microchip.
+Besides the wifi chip it also contains an SD card reader.
```

Comment:
"The shield also provides a micro SD card socket"

Reply:
Text changed. Next lines changed too. (reader -> socket)